payments-core/src/main/java/com/stripe/android/model/ConfirmPaymentIntentParams.kt

```diff
@@ -129,7 +131,11 @@ constructor(
 
     internal val paymentMethodCode: PaymentMethodCode? = paymentMethodCreateParams?.code,
 
-    @get:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) val radarOptions: RadarOptions? = null
+    @get:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) val radarOptions: RadarOptions? = null,
+
+    @get:VisibleForTesting
+    @get:RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    val clientAttributionMetadata: ClientAttributionMetadata? = null,
```

Comment:
Can this be internal? Or even private? Why do we need to expose it for testing? The only thing we really care about is that it renders to "json" correctly.